tests/partners/test_integration.py

```diff
@@ -1,17 +1,19 @@
 import copy
 import uuid
 
-from base import PartnerTestCase
 from sitewit.services import SitewitService
 
+from .base import PartnerTestCase
+
 
 def deep_sort(d):
     """Sort all lists inside given dict, at all levels."""
     dict_copy = copy.deepcopy(d)
 
     for key, value in d.items():
         if isinstance(value, list):
-            dict_copy[key] = sorted(value)
+            dict_copy[key] = sorted(value, key=lambda item: (
+                tuple(item.items()) if isinstance(item, dict) else item))
```

Comment:
But on line 14 it explicitly checks that value if a list's instance. Does the dict inherit from the list or vice versa?

Reply:
ah, item is not the value

Reply:
Did that line work prior your change?

Reply:
it works before with python2. in python3 it raises an exception because it can not compare dicts